packages/storage-plugin/internals/src/storage-key.ts

```diff
@@ -23,9 +23,9 @@ export type StorageKey = string | StateClass | StateToken<any> | KeyWithExplicit
 
 /** This symbol is used to store the metadata on state classes. */
 const META_OPTIONS_KEY = 'NGXS_OPTIONS_META';
-export function exctractStringKey(storageKey: StorageKey): string {
+export function ɵexctractStringKey(storageKey: StorageKey): string {
```

Comment:
Let's fix the spelling of this API before we expose it.
```suggestion
export function ɵextractStringKey(storageKey: StorageKey): string {
```